app/src/test/java/com/github/se/travelpouch/model/events/EventRepositoryTest.kt

```diff
@@ -0,0 +1,127 @@
+package com.github.se.travelpouch.model.events
+
+import android.os.Looper
+import androidx.test.core.app.ApplicationProvider
+import com.google.android.gms.tasks.Tasks
+import com.google.firebase.FirebaseApp
+import com.google.firebase.Timestamp
+import com.google.firebase.firestore.CollectionReference
+import com.google.firebase.firestore.DocumentReference
+import com.google.firebase.firestore.DocumentSnapshot
+import com.google.firebase.firestore.FirebaseFirestore
+import com.google.firebase.firestore.QuerySnapshot
+import junit.framework.TestCase.fail
+import org.hamcrest.CoreMatchers.`is`
+import org.hamcrest.MatcherAssert.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mock
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.`when`
+import org.mockito.MockitoAnnotations
+import org.mockito.kotlin.any
+import org.mockito.kotlin.timeout
+import org.mockito.kotlin.verify
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.Shadows.shadowOf
+
+@RunWith(RobolectricTestRunner::class)
+class EventRepositoryTest {
+  @Mock private lateinit var mockFirestore: FirebaseFirestore
+  @Mock private lateinit var mockDocumentReference: DocumentReference
+  @Mock private lateinit var mockCollectionReference: CollectionReference
+  @Mock private lateinit var mockDocumentSnapshot: DocumentSnapshot
+  @Mock private lateinit var mockToDoQuerySnapshot: QuerySnapshot
+
+  private lateinit var eventRepositoryFirestore: EventRepositoryFirebase
+
+  val event =
+      Event(
+          "1",
+          EventType.NEW_DOCUMENT,
+          Timestamp(0, 0),
+          "eventTitle",
+          "eventDescription",
+          null,
```

Comment:
No, it is just not relevant. Because this is only a String that is the uid of the participant. In this case, giving a random string or putting it to null does not change anything.

Reply:
That make sense. Thank you